### PR TITLE
Added navigation button, resolves #258

### LINKED ISSUE #258: Frontend - Add navigation Back button
Add navigation Back button to `Create new monitor` page.

image

### PR DESCRIPTION
**Pull request by danielcj2**
Added button to navigate back to `/monitors` from `/monitors/create`.

### CODE REVIEW

File: Client/src/Pages/CreateNewMonitor/index.jsx [hunk 1]
```diff
@@ -326,7 +341,7 @@ const CreateNewMonitor = () => {
                 MenuProps={{
                   PaperProps: {
                     style: {
-                      marginTop: "10px"
+                      marginTop: "10px",
```
Comment (ajhollid) [inline Client/src/Pages/CreateNewMonitor/index.jsx:344]: trailing comma ftw :joy:

**Review by ajhollid — COMMENTED** [review]
:+1:

**Review by ajhollid — APPROVED** [review]
:+1:

File: Client/src/Pages/CreateNewMonitor/index.jsx (hunk 1 above)
Comment (Skorpios604) [reply Client/src/Pages/CreateNewMonitor/index.jsx:344]: 😂😂

**Review by Skorpios604 — APPROVED** [review]
Looks good!

File: Client/src/Pages/CreateNewMonitor/index.jsx (hunk 1 above)
Comment (danielcj2) [reply Client/src/Pages/CreateNewMonitor/index.jsx:344]: Sniped it from 10 miles away 😂

**Pull request merged**